### DISCUSSION TITLE
Disconnecting Devices

### DISCUSSION THREAD

**Discussion by ovendylan** [dotnet-bluetooth-le/dotnet-bluetooth-le#851] [Q&A]
Why does the adapter appear to hold the connection hostage in the background somewhere? Using `await adapter.DisconnectDeviceAsync(dvc);` appears useless as the device may change state but subsequent scans end up empty because the device is still no longer available. If I want the device to appear available for scans I have to reset the Bluetooth on the phone.

**Comment by axa88** [comment]
What you _might_ be experiencing is various problems related to this on WinUI #850

But regardless, Im sure you could get better answers if you provide basic information such as:
- which platform are you having a problem with?
- what version of the lib are you using?

**Reply by ovendylan** [reply]
Thanks for the feedback. I've only really tested this with the latest version of Android and I am using version 3.0.0  of the library. What I see is that after making an initial connection to my device and running the disconnect method, unless I toggle Bluetooth on/off the device remains hidden from my list of available devices.

**Reply by axa88** [reply]
Well again it helps to know which platform you're having the problem with. 
Arguably this never worked correctly on WinUI. 

Anything else and most likely be asked to file a bug. But you'll need all the details of your issue.
